common/eq_packet_structs.h

```diff
@@ -4545,7 +4545,7 @@ struct ItemVerifyReply_Struct {
 struct ItemRecastDelay_Struct {
 /*000*/	uint32	recast_delay;	// in seconds
 /*004*/	uint32	recast_type;
-/*008*/	uint32	unknown008;
+/*008*/	bool	ignore_casting_requirement; //Ignores recast times allows items to be reset?
```

Comment:
Does making this a bool cause unintended consequences with the byte size difference? (1 byte vs 4 byte)

Did we test a case where we're not ignoring casting requirements just to make sure it works that way as well

Reply:
Size difference was tested and it never mattered